### PR TITLE
IAV-2380 Occupational clinic

### PR DESCRIPTION
**Pull request by juhog**
Tested on multiple different test patients:
- One with the clinic data
- One with occupational data without clinic
- One without occupational data

### CODE REVIEW

File: src/routes/Profile/components/ProfileView.js [hunk 1]
```diff
@@ -7,17 +7,24 @@ import List from '../../../components/List';
...
+    const { locations, fetchStaticEntities } = this.props;
+
     this.props.fetchContactDetails();
     this.props.fetchOccupationalInfo();
     if (this.props.doctors.data === null && this.props.doctors.isLoading !== true) {
       this.props.fetchDoctors();
     }
+
+    if (locations === null) {
+      fetchStaticEntities();
```
Comment (lauriii) [inline src/routes/Profile/components/ProfileView.js]: Why are you fetching specialties here?

Comment (juhog) [inline src/routes/Profile/components/ProfileView.js]: It's the combined fetcher. That's what other routes do too.

Comment (lauriii) [inline src/routes/Profile/components/ProfileView.js]: Yes, but they require both data. This route does not require specialties so we should only request for locations

Comment (juhog) [inline src/routes/Profile/components/ProfileView.js]: Some routes needs just specialties AFAIK. And they still do the static entities fetch.

Comment (lauriii) [inline src/routes/Profile/components/ProfileView.js]: IMHO thats a bug and should be fixed in follow up PR's

Comment (juhog) [inline src/routes/Profile/components/ProfileView.js]: I gotta stop for today so here are the options:
1) Fix all the static entity situations in a follow up and close this PR
2) Fix the profile view situation in this PR by someone before Friday morning meeting
3) Fix the profile view situation in this PR later on which means that this issue will be left open

**Pull request merged**